jest.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
 
   // An array of glob patterns indicating a set of files for which coverage information
   // should be collected
-  // collectCoverageFrom: undefined,
+  collectCoverageFrom: ['client/**/*.{js,jsx}'],
```

Comment:
I was playing with this as well, but it seemed to do the same thing when it was undefined.

Reply:
oh well, now it is more explicit.